app/views/passenger_mailer/notify_active.haml

```diff
@@ -0,0 +1,9 @@
+Dear #{@passenger.name},
+
+Your account with the Special Transportation Service has been activated. You can receive rides #{@passenger.rides_expire ? ('until ' + @passenger.rides_expire.strftime("%m/%d/%Y")) : 'permanently'}.
```

Comment:
One small nitpick: "Permanent" passengers can't actually receive rides _forever_. They can receive them as long as they meet the requirements. So I think I would replace 'permanently' with 'until further notice'